code/modules/reagents/reagent_containers/food/snacks/meat.dm

```diff
@@ -91,3 +91,15 @@
 	name = "snow strider meat"
 	desc = "A slab of nav'twir's meat, an animal native from Adhomai."
 	icon_state = "adhomai_meat"
+
+/obj/item/weapon/reagent_containers/food/snacks/meat/mice
+	name = "mice meat"
+	icon_state = "chickenbreast"
+	desc = "You have reached the epitome of poor, eating the poor mice of the station."

```

Comment:
The wording here is rather awkward and redundant; I suggest "You have reached the epitome of poorness: eating the station's vermin"